src/vinyldns/batch_change.py

```diff
@@ -176,6 +240,13 @@ def __init__(self, user_id, user_name, comments, created_timestamp,
         self.status = status
         self.id = id
         self.owner_group_id = owner_group_id
+        self.owner_group_name = owner_group_id
```

Comment:
`self.owner_group_name = owner_group_name`

Reply:
do i keep messing this up?